solution/src/main/openapi/solution.yaml

```diff
@@ -1119,12 +1232,41 @@ components:
         parameters:
           type: array
           description: An ordered list of Run Template Parameters
+          default: []
           items:
             type: string
             description: A Run Template Parameter id
       required:
         - id
 
+    RunTemplateParameterGroupUpdateRequest:
+      type: object
+      description: A Parameter Group Update Request for a Run Template
+      properties:
+        description:
+          type: string
+          description: A description of the parameter group
+        labels:
+          $ref: '#/components/schemas/TranslatedLabels'
+        isTable:
+          type: boolean
+          default: false
```

Comment:
There should not be default values on UpdateRequest fields, otherwise this means that not providing it explicitly will update them to their default value instead of keeping their value.
Same for `parameters`.